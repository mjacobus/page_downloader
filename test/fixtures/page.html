<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" />
    <title>Some Title</title>
    <script src="http://example.com/file.js"></script>
    <link href="http://example.com/file.css" rel="stylesheet" />
</head>
<body>
  <h1>Some Content</h1>
</body>
</html>
