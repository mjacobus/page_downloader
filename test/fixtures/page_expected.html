<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="UTF-8">
    <title>Some Title</title>
    <script>function foo() {}
</script>
    
<style>.foo { color: red; }
</style>
</head>
<body>
  <h1>Some Content</h1>
</body>
</html>
